fiftyone/utils/torch.py

```
@@ -2716,6 +2712,23 @@ def _parse_inputs(
         return image_paths, sample_ids, patch_edges, patches
 
 
+def _stack_transformed_patches(img_patches, use_numpy):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_stack_transformed_patches`

(ANN202)